lib/compute/agent-nodes.ts

```diff
@@ -234,13 +219,13 @@ export class AgentNodes {
     this.WINDOWS2019_X64_GRADLE_CHECK = {
       agentType: 'windows',
       customDeviceMapping: '/dev/sda1=:300:true:::encrypted',
-      workerLabelString: 'Jenkins-Agent-Windows2019-X64-C524xlarge-Single-Host',
-      instanceType: 'C524xlarge',
+      workerLabelString: 'Jenkins-Agent-Windows2019-X64-M58xlarge-Single-Host',
+      instanceType: 'M58xlarge',
```

Comment:
Whey are we changing the instance type, is C5 type not stable ?

Reply:
Yes, c524large is 1:2 cpu:mem while m58xlarge is 1:4 cpu:mem, which linux has already replace with.
Runs better, more stable, and less resource.

* #321